.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-6.2.1-SNAPSHOT
\ No newline at end of file
+6.3.0-SNAPSHOT
```

Comment:
We need to pin this version in the script too. I will do this on Monday, don't worry about it and enjoy the time off.